lib/PCP-server-Ruby-SDK/api/base_api_client.rb

```diff
@@ -0,0 +1,105 @@
+require 'net/http'
```

Comment:
Sollten den Ordner auch `endpoints` nennen, wie in den anderen SDKs

Reply:
You mean renaming the api directory to endpoints? Sure